src/main/java/com/epam/lab/paymentsystem/utility/AddTestData.java

```diff
@@ -0,0 +1,137 @@
+
+package com.epam.lab.paymentsystem.utility;
+
+import com.epam.lab.paymentsystem.dao.ConnectionPool;
+import javax.naming.NamingException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+
+class AddTestData {
+
+    public static Connection connection;
+
+    private final static List<String> USERS = new LinkedList<>();
+    private final static List<String> ACCOUNT = new LinkedList<>();
+    private final static List<Boolean> CARDS = new LinkedList<>();
+    private final static String SQL_USERS = "INSERT INTO users (login, passwd, id_role, user_name)" + " VALUES (?,?,?,?)";
+    private final static String SQL_ACCOUNTS = "INSERT INTO accounts (id_user, amount, is_active)" + " VALUES (?,?,?)";
+    private final static String SQL_CARDS = "INSERT INTO cards (id_user, id_account, is_active)" + " VALUES (?,?,?)";
+    private final static String SQL_ID_USER = "SELECT * FROM USERS WHERE login = ?";
+    private final static String SQL_ID_ACCOUNT = "SELECT * FROM ACCOUNTS WHERE id_user = ?";
+    private final static String SQL_ID_ROLE_USER = "SELECT * FROM USERS WHERE id_role = ?";
+
+    private static final int DATA_COUNT = 5;
+
+    private static void createData() {
+        USERS.add("zorita,1234,1,Zoy");
```

Comment:
You may use smth like that (for better redability):
```
Users.addAll(
       Collections.asList(
                          "zorita,1234,1,Zoy",
                          "chaim,qwer,2,Chester",
                          ...
                          )
);
```

Reply:
Ok, I used Arrays.asList()